lib/makara/connection_wrapper.rb

```diff
@@ -38,25 +38,25 @@ def _makara_shard_id
       @config[:shard_id]
     end
 
-    # has this node been blacklisted?
-    def _makara_blacklisted?
```

Comment:
Is it necessary to provide some kind of backward compatible support for these old methods? I would have assumed "yes" because they are public methods, but the `_` prefixes in the method names indicate to me that it is not intended for users to call or override these methods themselves.